README.md

```diff
@@ -4,7 +4,7 @@ This is modified version of script written by Andrew Ferguson: https://tardis.ed
 
 # Install
 
-``pip3 install echo360download``
+``pip3 install git+https://github.com/aalok-sathe/echo360download``
```

Comment:
I will republish the package so this is not really necessary..

Reply:
right, not sure why it also bunched all my future commits into this